metadata.md

```diff
@@ -89,11 +92,11 @@ Results for a measurement of a single property of SARS-CoV-2 wastewater test, fo
     -   `typeOther`: Other measures
 
 -   **measureTypeOther**: Description for other type of measurement type. See `measurementUnit`.
-
+ - **measureIndex**: index if the measurement was taken multiple times (int)
```

Comment:
change 'index' to 'Index' capital. 

A general note that there are small formatting differences in this PR. We may want to identify an MD flavour. I've increasingly using RStudio 1.4 MD editor which suggests formatting changes.

Reply:
I've been using https://stackedit.io/. I did not even know RStudio had and MD editor. It feels like it does everything now.